packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -97,15 +97,35 @@ export interface ICreateContainerMetadata {
 	createContainerTimestamp?: number;
 }
 
+/** @see IGCMetadata.gcFeatureSupportInfo */
+export interface GcFeatureSupportDimensions {
+	/**
+	 * Gives a way for an app to disqualify old files from GC Tombstone enforcement
+	 * Provided via Container Runtime Options
+	 */
+	appTombstoneReadiness?: number;
```

Comment:
I would suggest just using "tombstone" to indicate this is for the tombstone feature. Are you expecting there to be different properties for application vs FF?

Reply:
Went with `tombstoneGeneration` after discussion